internal/utils/common.go

```diff
@@ -185,6 +185,7 @@ func GetState() string {
 	return filepath.Join("/dev/disk/by-label/", label)
 }
 
+// TODO: This is doing the same thing as DetectUKIboot in kairos-sdk. Keep both?
```

Comment:
I wuld remove it and use the sdk directly, much nicer to have it just in one place!

Reply:
I made this function call the one in kairos-sdk (because the one in kairos-sdk expects a cmdline anyway). At least the "checking" logic will be in on place.